test/e2e_node/util.go

```diff
@@ -472,9 +472,11 @@ func toCgroupFsName(cgroupName cm.CgroupName) string {
 // to make the kernel reclaim memory in the allocatable cgroup
 // the time to reduce pressure may be unbounded, but usually finishes within a second
 func reduceAllocatableMemoryUsage() {
```

Comment:
this method was introduced as a reaction on flakes https://github.com/kubernetes/kubernetes/pull/60900/. I agree, simply dropping it for cgroupv2 sounds like a reasonable idea.

Can you please update a method comment to indicate that this method does nothing for the cgroupv2. I would even suggest to rename the method to `reduceAllocatableMemoryUsageIfCgroupv1`

Looking at https://www.spinics.net/lists/linux-mm/msg227679.html and your comment https://github.com/kubernetes/kubernetes/issues/107062#issuecomment-995413273 I feel like we may want to watch for flakes instead of trying memory.high solution.

Reply:
Let's rename it.

Once we have a solution for cgroup v2. We can rename it back and add the solution.